### DISCUSSION TITLE
Local Usage Issue

### DISCUSSION THREAD

**Discussion by faridhaghgooyan** [UniSharp/laravel-filemanager#1168] [Q&A]
Hi, 
I have a problem with local usage. As you know, some files like jquery, font awesome,bootstrap , ... are loaded from CDN.
I changed the links directly, but isn't there a better solution?

Regards,
Farid

**Comment by streamtw** [comment]
A total frontend refactoring with frameworks like Vue.js can achieve this, since webpack will generate only one single js file, makes it easier to determine or change if it is going to be served from local or CDN. But it will require a lot of time and work.

I am sorry to say that for now, the only way to achieve your requirement is to download all these js files, and then change the links in the `index.blade.php` file.
